substrate/client/consensus/manual-seal/src/lib.rs

```diff
@@ -236,7 +236,7 @@ pub async fn run_instant_seal<B, BI, CB, E, C, TP, SC, CIDP, P>(
 	// instant-seal creates blocks as soon as transactions are imported
 	// into the transaction pool.
 	let commands_stream = pool.import_notification_stream().map(|_| EngineCommand::SealNewBlock {
-		create_empty: false,
+		create_empty: true,
```

Comment:
Is this intended?

Reply:
Yeah, or else you could not generate empty blocks in instant/manual seal which is rather inconvenient, especially if you want to not have something like `timestamp`.